cli/js/timers.ts

```diff
@@ -15,13 +16,8 @@ interface Timer {
   scheduled: boolean;
 }
 
-// We'll subtract EPOCH every time we retrieve the time with Date.now(). This
-// ensures that absolute time values stay below UINT32_MAX - 2, which is the
-// maximum object key that EcmaScript considers "numerical". After running for
-// about a month, this is no longer true, and Deno explodes.
-// TODO(piscisaureus): fix that ^.
+// We'll subtract EPOCH every time we retrieve the time with Date.now().
```

Comment:
I think that with this patch you could remove EPOCH altogether.

Reply:
Yeah makes sense